hudi-common/src/main/java/org/apache/hudi/expression/Literal.java

```diff
@@ -80,22 +85,9 @@ public static <V> Literal from(V value) {
         + value.getClass().getName() + " to Literal");
   }
 
+  @Getter
```

Comment:
No need to add this as the `@Getter` is annotated to the class.

Reply:
Good catch! Missed this